packages/lib/src/vis/shared/Canvas.tsx

```diff
@@ -0,0 +1,27 @@
+import { Canvas as R3FCanvas } from '@react-three/fiber';
+import type { PropsWithChildren } from 'react';
+
+interface Props {
+  className?: string;
+  orthographic?: boolean;
+}
+
+function Canvas(props: PropsWithChildren<Props>) {
+  const { className, orthographic, children } = props;
+  return (
+    <R3FCanvas
+      className={className}
+      orthographic={orthographic}
+      linear // disable automatic color encoding and gamma correction
+      flat // disable tone mapping
+      frameloop="demand" // disable game loop
+      dpr={[1, 3]} // https://discoverthreejs.com/tips-and-tricks/#performance
+      resize={{ debounce: { scroll: 20, resize: 200 }, scroll: false }} // https://github.com/pmndrs/react-three-fiber/discussions/1906
+    >
+      <ambientLight />
+      {children}
+    </R3FCanvas>
+  );
+}
```

Comment:
New component `Canvas` to be used in both 2D and 3D vis with the appropriate props set.

Reply:
Coming back to this, I think this is very well worth exporting, as experimental or not, whichever you think. If we do so, perhaps we should actually swap the naming - i.e. export it in the lib as `R3FCanvas` and use the default naming of R3F `Canvas` component internally.